src/extension.ts

```diff
@@ -16,7 +16,7 @@
  */
 
 import * as vscode from 'vscode';
-import { extensionFunctions, initializeContext } from './extensionFunctions';
+import * as extensionFunctions from './extensionFunctions';
 import * as commandConstants from './extensionFunctions';
```

Comment:
there are 2 similar imports with different names. isn't better to use a single one?

Reply:
good catch